### DISCUSSION TITLE
Root Privilege in Enterprise instance

### DISCUSSION THREAD

**Discussion by vignesh-clustrex** [community/community#124938] [Enterprise] [answered]
### Body

I need to fetch all images inside git enterprise instance, I am getting access denied while changing directory, how can I get root privilege?

**Comment by gitstua** [comment, marked as answer]
You may be able to **read** data using the [SSH Admin Shell](https://docs.github.com/en/enterprise-server@3.12/admin/administering-your-instance/administering-your-instance-from-the-command-line/accessing-the-administrative-shell-ssh) however in most cases you should be using git to download files

Be [careful](https://docs.github.com/en/enterprise-server@3.12/admin/administering-your-instance/administering-your-instance-from-the-command-line/accessing-the-administrative-shell-ssh#access-limitations-for-the-administrative-shell) as you could break the appliance if you update things

**Comment by vignesh-clustrex** [comment]
Do you have any idea where the images are stored inside the instance?

**Reply by gitstua** [reply]
Sorry - I don't know that. Which images are you looking for?
